lep/DRAFT/011_Performance_Monitor_Goals_and_Metrics.md

```diff
@@ -0,0 +1,85 @@
+#LEP 011 - Performance Monitor Goals and Metrics
+
+Date:   March 24, 2015
+
+Status: Draft
+
+Author: fffw
+
+## Abstract
+
+To continuously monitor the performance of Lantern network, we need to define dimensions and metrics clearly, and collect data to fulfill the goal at both given and get mode.
+
+We must also keep user's anonymity.
+
+## Proposal
+
+The output, is a multi-dimensional model which can pivot from any dimension. The presentation solution is TBD, hopefully there are existing services or open source projects.
```

Comment:
Who is the audience of the presentation?  We originally made a public dashboard because we actually wanted to make Lantern performance and availability statistics open to the public - http://lantern-dashboard.s3-website-us-east-1.amazonaws.com/

If we're just interested in doing something for internal use, [Tableau](http://www.tableau.com/) is a well-respected BI solution that we should be able to integrate with BigQuery to create some canned reports.